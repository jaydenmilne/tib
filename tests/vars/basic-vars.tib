1 -> X
X
1 + X -> X

"burrito" -> Y
1.49323 -> Z
Z + X -> P

Y = "burrito"