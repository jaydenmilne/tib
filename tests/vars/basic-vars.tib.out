1
1
2
"burrito"
1.49323
3.49323
1
